apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -84,8 +84,19 @@ type ContourConfigurationSpec struct {
 
 	// Tracing defines properties for exporting trace data to OpenTelemetry.
 	Tracing *TracingConfig `json:"tracing,omitempty"`
+
+	// FeatureFlags defines toggle to enable new contour features.
+	// available toggles are
```

Comment:
nit:
```suggestion
	// Available toggles are:
	// useEndpointSlices - Configures contour to fetch endpoint data
	// from k8s EndpointSlices instead of the default behavior which reads
	// endpoint data from Endpoint resources.
```